### PR TITLE
Improvements authorization

### PR DESCRIPTION
**Pull request by danielpassos**
Some improvements for the Authorization screen/activity

### CODE REVIEW

**Comment by danielpassos** [comment]
@anuj1708 Can you make sure I did not break anything?

**Comment by openwarrior** [comment]
Work in progress ;)

**Comment by openwarrior** [comment]
It is working fine :+1:

File: src/main/res/layout/activity_authorization.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
 
     Copyright 2015 Red Hat, Inc. and/or its affiliates
```
Comment (openwarrior) [inline src/main/res/layout/activity_authorization.xml]: 2015 -> 2015-2016
:)

**Comment by danielpassos** [comment]
Build fixed!

**Comment by pilhuhn** [comment]
Thanks @danielpassos

**Pull request merged**